_gsocproposals/2021/proposal_RooUnfold.md

```diff
@@ -0,0 +1,37 @@
+---
+project: RooUnfold
+title: RooUnfold - Efficient deconvolution using state of the art algorithms
+layout: gsoc_proposal
+year: 2021
+organization:
+  - Tufts
+  - CERN
+---
+
+## Description
+[RooUnfold](https://roounfold.web.cern.ch) is a powerful modular analysis tool is for the unfolding (AKA "deconvolution" or "unsmearing") of distributions.
+Unfolding is the statistical process in which the 'true' underlying processes that govern the interactions of fundamental forces of the universe are estimated from both simulated and measured data.
+To facilitate these data intensive analyses, RooUnfold is built on the [ROOT](https://root.cern/) Software Framework and is used in the publication of hundreds of LHC analyses every year.
+RooUnfold was designed to provide a common interface for a number of analysis algorithms and was [recently upgraded](https://arxiv.org/abs/1910.14654) to leverage the power of the ROOT statistical modelling package RooFit in order to provide a statistical comparison of the various algorithms.
+
+This project aims to address obscelescance in the underlying RooUnfold implementation that would allow the RooUnfold package to act as a lightweight member class to ROOT whilst retaining it’s functionality and the ability for users to define new algorithms including machine learning based solutions, and data intensive tasks. 
```

Comment:
obscelescance => obsolescence ?
retaining it’s functionality => retaining its functionality ?

Reply:
fixed

Reply:
Thank you, Vincent, but could you put entries sorted in last name alphabetic order?

Reply:
ah ok yes. Very democratiic. I'll do that now.

Reply:
in mentors.md also?

Reply:
Thanks! 
But 2 files are missing too:
- a file describing RooUnfold project (`_gsocprojects/2021/project_ RooUnfold.md`)
- a file describing Tufts organization (`_gsocorgs/2021/tufts.md`)

Reply:
ok I think that's done! Thanks for all your effort on this!